### DISCUSSION TITLE
How can we hide fields in generations

### DISCUSSION THREAD

**Discussion by vuletuanbt** [dotansimha/graphql-code-generator#7995] [Q&A]
We have tried to integrate your package into our system. I worked well in simple cases. Now we want to hide some fields in some cases, we don't want to return them to the front end. So, how can we archive it?.

**Comment by saihaj** [comment]
Sorry can you elaborate more on what you are trying to achieve? This just generates types for your schema and is not something that is executed or _sent_  to client.
